src/ImageService.Broker/Consumers/CreateImagesConsumer.cs

```diff
@@ -8,238 +11,184 @@
 using LT.DigitalOffice.Models.Broker.Requests.Image;
 using LT.DigitalOffice.Models.Broker.Responses.Image;
 using MassTransit;
-using System;
-using System.Collections.Generic;
-using System.Threading.Tasks;
 
 namespace LT.DigitalOffice.ImageService.Broker.Consumers
 {
-    public class CreateImagesConsumer : IConsumer<ICreateImageRequest>
+  public class CreateImagesConsumer : IConsumer<ICreateImageRequest>
```

Comment:
update models to 37 version in common.configuration.targets and edit this and other from ICreateImageRequest to ICreateImagesRequest

Reply:
Made in a different task